deploy/firstboot/centos7-kvm.sh

```diff
@@ -93,3 +93,8 @@ STP=yes" > /etc/sysconfig/network-scripts/ifcfg-cloudbr1
 # Reboot
 reboot
 
+# Keep the script running until the reboot happens
+while :
+do
+  # loop infinitely
```

Comment:
Yes. Otherwise this one will exit, and the post boot script will start and exit before the VM has had the chance to reboot. Ultimately will make the deploy script exit altogether before the machine has rebooted.